src/check/canonicalize.zig

```diff
@@ -205,14 +212,15 @@ fn bringImportIntoScope(
         const exposed = self.parse_ir.store.getExposedItem(exposed_idx);
         switch (exposed) {
             .lower_ident => |ident| {
+
+                // TODO handle `as` here using an Alias
+
                 if (self.parse_ir.tokens.resolveIdentifier(ident.ident)) |ident_idx| {
-                    if (ident.as) |as_| {
-                        if (self.parse_ir.tokens.resolveIdentifier(as_)) |alias_idx| {
-                            _ = self.scope.levels.introduce(.alias, .{ .scope_name = ident_idx, .alias = alias_idx });
-                        }
-                    } else {
-                        _ = self.scope.levels.introduce(.ident, .{ .scope_name = ident_idx, .ident = ident_idx });
-                    }
+                    _ = ident_idx;
+
+                    // TODO Introduce our import
+
+                    // _ = self.scope.levels.introduce(gpa, &self.can_ir.env.idents, .ident, .{ .scope_name = ident_idx, .ident = ident_idx });
```

Comment:
I'm leaving imports for a later PR. I've just commented most of this out for now.